docs/language/learn-ql/writing-queries/introduction-to-queries.rst

```diff
@@ -128,6 +128,24 @@ Select clauses for metric queries (``@kind metric``) consist of two 'columns', w
 - ``element``: a code element that is identified by the query, which defines where the alert is displayed.
 - ``metric``: the result of the metric that the query computes.
 
+Viewing the standard CodeQL queries
+***********************************
+
+One of the easiest ways to get started writing your own queries is to modify an existing query. To see view the standard CodeQL queries, or to try out other examples, visit the `CodeQL <https://github.com/semmle/ql>`__ and `CodeQL for Go <https://github.com/github/codeql-qo>`__ repositories on GitHub. 
+
+You can also find examples of queries developed to find security vulnerabilities and bugs in open-source software projects on the `GitHub Security Lab website <https://securitylab.github.com/research>`__ and in the associated `repository <https://github.com/github/security-lab>`__.
+
+Contributing queries
+********************
+
+Contributions to the standard queries and libraries are very welcome---see our `contributing guidelines <https://github.com/Semmle/ql/blob/master/CONTRIBUTING.md>`__ for further information.
+If you are contributing a query to the open source GitHub repository, writing a custom query for LGTM, or using a custom query in an analysis with the CodeQL CLI, then you need to include extra metadata in your query to ensure that the query results are interpreted and displayed correctly. See the following topics for more information on query metadata:
+
+-  :doc:`Query metadata reference <query-metadata>`
+-  `Query metadata style guide on GitHub <https://github.com/Semmle/ql/blob/master/docs/query-metadata-style-guide.md>`__
+
+Query contributions to the open source GitHub repository may also have an accompanying query help file to provide information about their purpose for other users. For more information on writing query help, see the `Query help style guide on GitHub <https://github.com/Semmle/ql/blob/master/docs/query-help-style-guide.md>`__ and the :doc:`Query help reference <query-help>`.
```

Comment:
Here also "open source" without a hyphen is used.
Is this intentional or am I missing a subtle difference in the usage?

Reply:
> am I missing a subtle difference in the usage?

Nope, we have just been inconsistent in the past 😄. I'll update this to follow the MS style guide convention: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source

Reply:
Nope, we have just been inconsistent in the past 😄 
Really we should be following the advice in this brand guide: https://brand.github.com/content/grammar